test/mitmproxy/addons/test_blocklist.py

```diff
@@ -43,6 +43,27 @@ def test_block(self, filter, status_code):
             else:
                 assert not f.response
 
+    @pytest.mark.parametrize(
+        "filter,should_block",
+        [
+            ("/~u AccountsSignInUi/444", True),
+            ("/~u accountssigninui/444", True),
+            ("/~u .ccounts.ign.n.i/444", True),
+            ("/~u example.com/444", False),
```

Comment:
Should we just extend the test above with another parametrization? :)

Reply:
yes sure! the problem was in the url so I thought to add a different test but it makes perfect sense